mbofui/app/js/controllers.js

```diff
@@ -342,6 +348,16 @@ ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter
       'event': targetEvent,
       'respondent': respondent
     };
+    if(data.intention === 'declined'){
+      if (window.confirm("Are you sure you do not want to go?")) {
```

Comment:
Could make this into a string variable if we plan on using this in another location... but I don't think we need to do so.